tests_v2/test_background_workspace_idle_thread_footprint.py

```
@@ -0,0 +1,219 @@
+class ProcessMetrics:
+    thread_count: int
+    physical_footprint_bytes: int
+
+
+def _must(cond: bool, msg: str) -> None:
+    if not cond:
+        raise cmuxError(msg)
+
+
+def _run(cmd: list[str]) -> str:
+    proc = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)